scripts/mbedtls_dev/asymmetric_key_data.py

```diff
@@ -49,14 +49,80 @@ def construct_asymmetric_key_data(src) -> Dict[str, Dict[int, bytes]]:
 ## pairs consist of a private key and its public key.
 #pylint: disable=line-too-long
 ASYMMETRIC_KEY_DATA = construct_asymmetric_key_data({
+    'ECC(PSA_ECC_FAMILY_SECP_K1)': {
+        192: ("297ac1722ccac7589ecb240dc719842538ca974beb79f228",
+              "0426b7bb38da649ac2138fc050c6548b32553dab68afebc36105d325b75538c12323cb0764789ecb992671beb2b6bef2f5"),
+        224: ("0024122bf020fa113f6c0ac978dfbd41f749257a9468febdbe0dc9f7e8",
+              "042cc7335f4b76042bed44ef45959a62aa215f7a5ff0c8111b8c44ed654ee71c1918326ad485b2d599fe2a6eab096ee26d977334d2bac6d61d"),
+        256: ("7fa06fa02d0e911b9a47fdc17d2d962ca01e2f31d60c6212d0ed7e3bba23a7b9",
+              "045c39154579efd667adc73a81015a797d2c8682cdfbd3c3553c4a185d481cdc50e42a0e1cbc3ca29a32a645e927f54beaed14c9dbbf8279d725f5495ca924b24d"),
+    },
     'ECC(PSA_ECC_FAMILY_SECP_R1)': {
+        225: ("872f203b3ad35b7f2ecc803c3a0e1e0b1ed61cc1afe71b189cd4c995",
```

Comment:
I don't think there's a 225-bit curve in [this family](https://arm-software.github.io/psa-api/crypto/1.1/api/keys/types.html#c.PSA_ECC_FAMILY_SECP_R1) - and also the 192 and 224 bit curves are missing.

Reply:
Right, 225 should have been 224, I probably mixed up K1 and R1 here.

Reply:
Fix incoming: https://github.com/Mbed-TLS/mbedtls/pull/8668